boxcli/build.go

```diff
@@ -12,8 +12,13 @@ import (
 	"go.jetpack.io/devbox/docker"
 )
 
+type buildCmdFlags struct {
+	config configFlags
+	docker *docker.BuildFlags
+}
+
 func BuildCmd() *cobra.Command {
-	flags := &docker.BuildFlags{}
+	flags := newBuildFlags()
 
 	command := &cobra.Command{
 		Use:   "build [<dir>]",
```

Comment:
Update `Use` to not have `[<dir>]` – same for any other commands where this is changing

Reply:
ah, yeah, I overlooked that

Reply:
Fixed